store/storetest/user_store.go

```diff
@@ -1715,22 +1715,22 @@ func testUserUnreadCount(t *testing.T, ss store.Store) {
 	p1.Message = "this is a message for @" + u2.Username
 
 	// Post one message with mention to open channel
-	store.Must(ss.Post().Save(&p1))
+	ss.Post().Save(&p1)
```

Comment:
I would expect a `require.Nil(t, err)` check here and all the other `store.Must` replacements in this file